blessedDeps.gradle

```diff
@@ -22,6 +23,7 @@ rootProject.ext {
             constraintLayout: "com.android.support.constraint:constraint-layout:$CONSTRAINT_LAYOUT_VERSION",
             espresso        : "com.android.support.test.espresso:espresso-core:$ESPRESSO_VERSION",
             javaPoet        : "com.squareup:javapoet:$JAVAPOET_VERSION",
+            kotlinPoet      : "com.squareup:kotlinpoet:$KOTLINPOET_VERSION",
```

Comment:
Same here. Mind adding a comment that says these should be kept in alphabetical order?